protocol/xmrmaker/net.go

```diff
@@ -110,44 +110,48 @@ func (inst *Instance) HandleInitiateMessage(
 		return nil, errOfferIDNotSet
 	}
 
-	// TODO: If this is not ETH, we need quick/easy access to the number
-	//       of token decimal places. Should it be in the OfferExtra struct?
-	err := coins.ValidatePositive("providedAmount", coins.NumEtherDecimals, msg.ProvidedAmount)
+	offer, offerExtra, err := inst.offerManager.GetOffer(msg.OfferID)
 	if err != nil {
 		return nil, err
 	}
 
-	offer, offerExtra, err := inst.offerManager.GetOffer(msg.OfferID)
+	var maxDecimals uint8 = coins.NumEtherDecimals
```

Comment:
```suggestion
	maxDecimals := coins.NumEtherDecimals
```

Reply:
`NumEtherDecimals` is not-typed constant. Would you prefer `maxDecimals := uint8(coins.NumEtherDecimals)` over the way it is now?

Reply:
I switched it and the other identical one to use a cast on the non-typed constant instead of a `var` declaration.